httpmw/Ip_cognito_auth_test.go

```diff
@@ -94,11 +108,42 @@ func TestCognitoIpAuth(t *testing.T) {
 	assert.Equal(http.StatusOK, w.Code)
 }
 
+func TestCognitoIpAuthTokenFails(t *testing.T) {
+	assert := assert.New(t)
+	ipRanges := "192.168.20.1-192.168.20.10"
```

Comment:
We can create a `const` called `authTestIPRanges` I see this value is repeated at least twice already.